src/west/commands/west-commands-schema.yml

```diff
@@ -4,51 +4,12 @@
 ##
 ## By convention, this file is named west-commands.yml.
 
-# The top-level of the file is a map with a 'west-commands:' key.  The
-# west-commands key's value is a sequence of "command descriptors".
-# Each command descriptor gives the location of a file containing west
-# commands, along with the names of the commands provided by the file
-# (and optionally the names of the classes which define them).
+# --------------------------------------------------------------
 #
-# Here is an example west-commands.yml file:
+#  DO NOT CHANGE THIS FILE WITHOUT UPDATING THE DOCUMENTATION!
 #
-# west-commands:
```

Comment:
I still think the example and description in the schema file is useful.
If this has been moved to Zephyr docs, it's fine with me, but if not, I think this is still valid and useful to people who are gonna create extension commands.

I see no description of this kind in:
https://docs.zephyrproject.org/latest/guides/west/index.html
https://docs.zephyrproject.org/latest/guides/west/repo-tool.html
https://docs.zephyrproject.org/latest/guides/west/build-flash-debug.html
https://docs.zephyrproject.org/latest/guides/west/west-not-found.html#west-apis

Instead of removing, could we have this information placed elsewhere ?
(Sorry if I have missed a Zephyr PR, tried to look for something related to west / docs but didn't find any)

Reply:
> We'll move this information into
the zephyr documentation.

I noticed this in the commit message, so I assume you're already working on that :)
So guess this answer my question, and you can ignore the comment.

Reply:
You didn't miss anything -- these docs aren't upstream yet. As you guessed from the commit message, I am almost ready to submit this docs PR:

https://github.com/mbolivar/zephyr/tree/west-docs-v1.14

The only thing left is to document `west sign`, which I am working on now. I need this PR to go in first, though, or the API autodoc directives will break the build.

Reply:
Great, just as guessed :)